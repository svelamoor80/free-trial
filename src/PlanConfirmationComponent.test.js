import React from 'react';
import renderer from 'react-test-renderer';

const props = {
    "match":
    {
        "path": "/plan/:plan",
        "url": "/Plan/29",
        "isExact": true,
        "params": { "plan": "29" }
    },
    "location": {
        "pathname": "/Plan/29",
        "myCustProps": {
            "title": "Entrepreneur",
            "planCharge": 29,
            "planSave": "$250",
            "selectPlanText": "Select Plan",
            "link": "/Plan",
            "switched": false
        },
        "search": "",
        "hash": "",
        "key": "wnq6x7"
    },
}
it('Toggles correctly', () => {
    const Plan = function (props) {
        return (
            `<div>
                <div class="planConfirmBody">
                    <div>'Thank you. You have selected Monthly plan of Entrepreneur at $29.'</div>
                    <a class='teikametricsRectangleBox' href='/'>
                        <span id='teikamtericsLabel'>Go Back</span>
                    </a>
                </div>
            </div>`
        )
    }

    const tree = renderer.create(<Plan />).toJSON();
    expect(tree).toMatchSnapshot();
});