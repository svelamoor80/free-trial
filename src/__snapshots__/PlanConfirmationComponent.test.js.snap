// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toggles correctly 1`] = `
"<div>
                <div class=\\"planConfirmBody\\">
                    <div>'Thank you. You have selected Monthly plan of Entrepreneur at $29.'</div>
                    <a class='teikametricsRectangleBox' href='/'>
                        <span id='teikamtericsLabel'>Go Back</span>
                    </a>
                </div>
            </div>"
`;
